unit_test/test_postgres_job_manager.py

```diff
@@ -0,0 +1,79 @@
+import unittest
```

Comment:
Moved these tests out of `test_main.py` to its own file as the src isn't located in`main.py`